src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div class="calendar calendar--day-of-week_sun calendar--month-length_29">
+        <div class="calendar__day calendar__day--number_1"></div>
```

Comment:
Don't add extra classes.
If you have to change main class, you'll have to change each and every one of them.
```suggestion
        <div class="calendar__day></div>
```